clj-refactor.el

```diff
@@ -221,6 +221,15 @@ won't run if there is a broken namespace in the project."
   :group 'cljr
   :type 'boolean)
 
+(defcustom cljr-ignore-libspecs
```

Comment:
thinking along replacing `libspecs` with `ns` but keeping `ignore` but don't have a really good one. perhaps `cljr-ns-to-ignore-patterns`?!